changelog/fragments/return-olm-install-error.yaml

```diff
@@ -0,0 +1,6 @@
+entries:
+  - description: >
+      `Status.HasInstalledResources()` now returns aggregated errors of all resource failures occurred while installing OLM. And made these errors visible to the user upon installation/get status failure.
```

Comment:
```suggestion
      `olm` and `run` subcommands will print aggregated resource errors when either OLM
      or an operator fail to install, respectively.
```

Reply:
Changed the description as suggested. Thanks @estroz.